### PR TITLE
SNOW-1011779: Suspend and resume commands for SPCS services.

### PR DESCRIPTION
**Pull request by sfc-gh-davwang**
### Pre-review checklist
   * [x] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [x] I've added or updated automated unit tests to verify correctness of my new code.
   * [x] I've added or updated integration tests to verify correctness of my new code.
   * [x] I've confirmed that my changes are working by executing CLI's commands manually.
   * [x] I've confirmed that my changes are up-to-date with the target branch.
   * [x] I've described my changes in the release notes.
   * [x] I've described my changes in the section below.

### Changes description
Added `spcs service suspend <name>` and `spcs service resume <name>` commands with tests.
resume_service
suspend_service

### CODE REVIEW

File: src/snowflake/cli/api/project/util.py [hunk 1]
```diff
@@ -12,8 +12,11 @@
 SINGLE_QUOTED_STRING_LITERAL_REGEX = r"'((?:\\.|''|[^'\n])+?)'"
 
 # See https://docs.snowflake.com/en/sql-reference/identifiers-syntax for identifier syntax
-UNQUOTED_IDENTIFIER_REGEX = r"(^[a-zA-Z_])([a-zA-Z0-9_$]{0,254})"
+UNQUOTED_IDENTIFIER_REGEX = r"([a-zA-Z_])([a-zA-Z0-9_$]{0,254})"
 QUOTED_IDENTIFIER_REGEX = r'"((""|[^"]){0,255})"'
```
Comment (sfc-gh-davwang) [inline src/snowflake/cli/api/project/util.py:16]: As far as I can see, all uses of UNQUOTED_IDENTIFIER_REGEX are used with re.fullmatch (as they should), so I believe this start-of-string character is unnecessary

File: tests/object/test_common.py [hunk 2]
```diff
@@ -6,6 +7,9 @@
 from typing import Tuple
 import pytest
 
+from unittest import mock
```
Comment (sfc-gh-yiwang) [inline tests/object/test_common.py:10]: are those imports required? I don't see the usage of them

**Review by sfc-gh-yiwang — DISMISSED** [review]
Overall LGTM

**Comment by sfc-gh-davwang** [comment]
Ignore the large diffs in test_help_messages.ambr for now, should go away once https://github.com/Snowflake-Labs/snowcli/pull/767 is merged.

File: tests/object/test_common.py (hunk 2 above)
Comment (sfc-gh-davwang) [reply tests/object/test_common.py:10]: good point, removed

File: tests/__snapshots__/test_help_messages.ambr [hunk 3]
```diff
@@ -1,30 +1,4 @@
 # serializer version: 1
-# name: test_help_messages[app.bundle]
-  '''
-                                                                                  
-   Usage: default app bundle [OPTIONS]
```
[the remark this answers is not in the document]
Comment (sfc-gh-davwang) [reply tests/__snapshots__/test_help_messages.ambr:5]: yep, see https://github.com/Snowflake-Labs/snowcli/pull/767

**Review by sfc-gh-turbaszek — DISMISSED** [review]

**Review by sfc-gh-yiwang — APPROVED** [review]

**Review by sfc-gh-turbaszek — APPROVED** [review]

**Pull request merged**